### ISSUE TITLE
IE11 Support?

### ISSUE THREAD

**Issue by kaabij** [szhsin/react-menu#13]
I hate to be the guy asking this... but does react-menu support IE11? I have tried it locally in a project but the menu disappears as soon as the menu button is clicked. In a CRA with just react-menu:

```javascript
import { Menu, MenuItem, MenuButton } from "@szhsin/react-menu/dist/index.js";
import "@szhsin/react-menu/dist/index.css";

function App() {
  return (
    <div>
       <Menu menuButton={<MenuButton>Open menu</MenuButton>}>
         <MenuItem>Test Item 1</MenuItem>
         <MenuItem>Test Item 2</MenuItem>
      </Menu>
    </div>
  );
}
```
This will render the button correctly, but once pressed, IE11 will output an error saying `Object doesn't support property or method 'includes'`. Since this can probably be resolved with a polyfill, I added:
```javascript
import "core-js/features/array/includes";
```
With that, the menu opens, but instantly closes right after and behaves radically.
<img src="https://user-images.githubusercontent.com/14207494/102145343-e6abf400-3e1b-11eb-8ca0-83c1fd1a2158.gif" width="220">

**Comment by szhsin** [comment]
Hi @kaabij , 
The library hasn't really been tested in IE11 yet. Will do an assessment of the issue, thanks for your feedback.

**Comment by kaabij** [comment]
Thanks @szhsin! I looked at it again and this time I specified `browserslist` in package.json to include `ie 11`, cleared `node_modules`, and re-ran. I'm not certain if I had some other configuration issue on my end but it appears to have resolved and react-menu works in IE11, though I have not tested all the features there yet. I'll report back if I find anything else.

**Comment by szhsin** [comment]
Cool, thanks @kaabij for all the testing and report. I'll do some further investigation into the IE11 support anyway.

**Comment by szhsin** [comment]
Resolved IE 11 issues in [v1.2.2](https://github.com/szhsin/react-menu/releases/tag/v1.2.2). Please note polyfills are still required; tested with [core-js](https://github.com/zloirock/core-js).
